apps/system-e2e/src/tests/islandis/service-portal/smoke/company-delegation.spec.ts

```diff
@@ -43,7 +43,42 @@ test.describe('Service portal', () => {
       waitUntil: 'domcontentloaded',
     })
 
+    const dashboard = page.getByTestId('service-portal-dashboard')
+
     // Assert
     await expect(findByRole('heading', companyName ?? '')).toBeVisible()
+    await expect(dashboard).toBeVisible()
+    await expect(await dashboard.locator('a').count()).toBeLessThan(10)
+  })
+
+  test('can view company data', async () => {
+    // Arrange
+    const page = await context.newPage()
+    const { findByRole } = helpers(page)
+    await page.goto('/minarsidur?locale=is&hide_onboarding_modal=true')
+
+    // Act
+    await page.locator('data-testid=user-menu >> visible=true').click()
+    await page.locator('role=button[name="Skipta um notanda"]').click()
+    const firstCompany = page.locator('role=button[name*="Prókúra"]').first()
+    await expect(firstCompany).toBeVisible()
+    const companyName = await firstCompany
+      .locator('.identity-card--name')
```

Comment:
[Would be better to avoid CSS classnames and use ARIA attributes. Or fallback on testid.](https://www.notion.so/System-E2E-testing-workshop-29c089c182844933b992015c4ef24791?pvs=4#11e4f2594bc140cfa95b1b33ba2452b7)